bftclient/include/bftclient/config.h

```diff
@@ -68,6 +68,7 @@ struct ClientConfig {
   RetryTimeoutConfig retry_timeout_config;
   std::optional<std::string> transaction_signing_private_key_file_path = std::nullopt;
   std::optional<concord::secretsmanager::SecretData> secrets_manager_config = std::nullopt;
+  std::optional<std::string> replicas_master_key_folder_path = "./replicas_rsa_keys";
```

Comment:
nit: Should we name path as: "./replicas_keys" or "./replicas_master_keys"

Reply:
we should, but as long as we work with rsa keys I prefer to specify this (to ease debugging and so)